client/my-sites/checkout/checkout-thank-you/index.tsx

```diff
@@ -559,19 +563,6 @@ export class CheckoutThankYou extends Component<
 			wasBulkDomainTransfer = isBulkDomainTransfer( purchases );
 		}
 
-		// this placeholder is using just wp logo here because two possible states do not share a common layout
-		if (
-			! purchases.length &&
-			! failedPurchases.length &&
-			! this.isGenericReceipt() &&
-			! this.props.selectedSite
-		) {
-			// disabled because we use global loader icon
-			/* eslint-disable wpcalypso/jsx-classname-namespace */
-			return <WordPressLogo className="wpcom-site__logo" />;
-			/* eslint-enable wpcalypso/jsx-classname-namespace */
-		}
-
```

Comment:
I totally don't see the necessity of this condition. Maybe it's something legacy.
More over, if we don't have `purchases`, `failedPurchases` and `is NOT a generic receipt` - then looks like "something went wrong" and instead of handling it properly we jut show endless WP logo w/o any explanation. In general it looks weird and I would remove it.